database/seedFunctions.js

```diff
@@ -0,0 +1,169 @@
+  const languages = [
+    'Arabic',
+    'French',
+    'Portuguese (European)',
+    'Chinese (Simplified)',
+    'Italian',
+    'Vietnamese',
+    'German',
+    'Russian',
+    'English',
+    'Hebrew',
+    'Spanish',
+    'Hindi',
+    'Japanese',
+    'Turkish',
+    'Gujarati',
+    'Polish',
+    'Persian',
+    'Kannada',
+    'Romanian',
+  ];
+  const numberOfLangsToUse = generateNumberWithinRange(0, languages.length);
+  for (let i = 0; i < numberOfLangsToUse; i++) {
+    const randomLanguageIndex = generateNumberWithinRange(0, languages.length);
+    chosenLanguages.push(languages.splice(randomLanguageIndex, 1)[0]);
+  }
+  return chosenLanguages;
+};
+
+const generateLearnerCareerOutcomes = () => {
+  const outcomes = [
+    {
+      icon: 'signpost',
+      pct: generateRandomPercentage(),
+      outcome: 'started a new career after completing these courses',
+    },
+    {
+      icon: 'briefcase',
+      pct: generateRandomPercentage(),
+      outcome: 'got a tangible career benefit from this course',
+    },
+    {
+      icon: 'money',
+      pct: generateRandomPercentage(),
+      outcome: 'got a pay increase or promotion',
+    },
+  ];
+
+  return outcomes;
+};
+
+const generateMetadata = () => {
+  const randomHours = generateNumberWithinRange(20, 200);
+  const subtitleLanguages = `Subtitles: ${generateLanguageList().join(', ')}`;
+  const icons = [
+    {
+      icon: 'certificate',
+      title: 'Shareable Certificate',
+      subtitle: 'Earn a Certificate upon completion',
+    },
+    {
+      icon: 'globe',
+      title: '100% online',
+      subtitle: 'Start instantly and learn at your own schedule',
+    },
+    {
+      icon: 'calendar',
+      title: 'Flexible Deadlines',
+      subtitle: 'Reset deadlines in accordance to your schedule',
+    },
+    {
+      icon: 'clock',
+      title: `Approx. ${randomHours} hours to complete`,
+      subtitle: '',
+    },
+    {
+      icon: 'speechbubble',
+      title: 'English',
+      subtitle: subtitleLanguages,
+    },
+  ];
+  return icons;
+};
+
+const generateWhatYouWillLearn = async () => {
+  const whatYouWillLearn = [];
+  for (let i = 0; i < 4; i++) {
+    const text = await generateFillerText({ sentences: 2 });
+    whatYouWillLearn.push(text);
+  }
+  return whatYouWillLearn;
+};
+
+const generateSkillsYouWillGain = async () => {
+  const skills = [];
+  const numOfSkills = generateNumberWithinRange(0, 10);
+  for (let i = 0; i < numOfSkills; i++) {
+    let skill = await generateFillerText({ sentences: 1 });
+    if (skill.split(' ').length > 4) {
+      const numOfWords = generateNumberWithinRange(2, 4);
+      skill = skill.split(' ').slice(0, numOfWords).join(' ');
+    }
+    skills.push(skill);
+  }
+  return skills;
+};
+
+const generateRecords = async (numToGenerate) => {
+  const records = [];
+  for (let i = 1; i < numToGenerate + 1; i++) {
+    console.log(`Creating record ${i}`);
+    const item = {
+      course_id: i, // 1 - 100
```

Comment:
What happens when you seeding multiple times ? Won't there be courses with same ID's always starting from 1-100